consensus/blake3pow/poem.go

```diff
@@ -121,3 +121,39 @@ func (blake3pow *Blake3pow) DeltaLogS(header *types.Header) *big.Int {
 	}
 	return big.NewInt(0)
 }
+
+func (blake3pow *Blake3pow) UncledLogS(block *types.Block) *big.Int {
+	uncles := block.Uncles()
+	totalUncledLogS := big.NewInt(0)
+	for _, uncle := range uncles {
+		// Verify the seal and get the powHash for the given header
+		err := blake3pow.verifySeal(uncle)
+		if err != nil {
+			continue
+		}
+		// Get entropy reduction of this header
+		intrinsicS := blake3pow.IntrinsicLogS(uncle.Hash())
+		totalUncledLogS.Add(totalUncledLogS, intrinsicS)
+	}
+	return totalUncledLogS
+}
+
+func (blake3pow *Blake3pow) UncledSubDeltaLogS(header *types.Header) *big.Int {
+	_, order, err := blake3pow.CalcOrder(header)
+	if err != nil {
+		return big.NewInt(0)
+	}
+	uncledLogS := header.UncledS()
+	switch order {
```

Comment:
This switch confuses me. "subXYZ" is a measurement in dom about the sub. It seems like each of these "subXYZ" measurements are stored in the sub though?

This is a minor lexicographical question.. the logic in each computation seems correct.

Reply:
It is stored in the sub because only it can calculate the values, and used in dom, similar to how we do the manifests